src/client/modules/common/components/index.jsx

```diff
@@ -1,3 +1,4 @@
 // eslint-disable-next-line import/prefer-default-export
 export { default as Button } from './Button';
 export { default as LayoutCenter } from './LayoutCenter';
+export { default as ClientOnly } from './ClientOnly';
```

Comment:
The export name here should be `clientOnly`, case is important. And the file name should be `clientOnly.jsx`